### DISCUSSION TITLE
How to center the header?

### DISCUSSION THREAD

**Discussion by PottiMc** [totaldebug/atomic-calendar-revive#1443] [Support] [answered]
I am trying to center the header of the calendar-card and I am not able to find a solution with card-mod. Maybe you can help me?

![image](https://github.com/totaldebug/atomic-calendar-revive/assets/39881308/504886a5-db1e-4949-9655-4d5b6b7a3f5a)

**Comment by dachico** [comment, marked as answer]
Hey Potti, you can target the header in your CSS file and apply text-align: center on it, or you can use flexbox on your header in CSS using display:flex, justify-content: center, align-content: center, hope that helped!
